### PR TITLE
vhs: fix #208002 add chromium dependency

### LINKED ISSUE #208002: vhs: fails to run chromium when running readme demo
### Steps To Reproduce
1. Run `vhs new demo.tape`
2. Run `vhs demo.tape`

### Expected behavior
demo.gif created with expected behavior from demo.tape

### Screenshots
```shell
$ vhs demo.tape                                        
File: demo.tape
[launcher.Browser]2022/12/27 14:12:42 try to find the fastest host to download the browser binary
[launcher.Browser]2022/12/27 14:12:42 check https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/1033860/chrome-linux.zip
[launcher.Browser]2022/12/27 14:12:42 check https://registry.npmmirror.com/-/binary/chromium-browser-snapshots/Linux_x64/1033860/chrome-linux.zip
[launcher.Browser]2022/12/27 14:12:42 check https://playwright.azureedge.net/builds/chromium/1033860/chromium-linux-arm64.zip
[launcher.Browser]2022/12/27 14:12:42 check result: Get "https://playwright.azureedge.net/builds/chromium/1033860/chromium-linux-arm64.zip": context canceled
[launcher.Browser]2022/12/27 14:12:42 check result: Get "https://registry.npmmirror.com/-/binary/chromium-browser-snapshots/Linux_x64/1033860/chrome-linux.zip": context canceled
[launcher.Browser]2022/12/27 14:12:42 Download: https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/1033860/chrome-linux.zip
[launcher.Browser]2022/12/27 14:12:42 Progress:
[launcher.Browser]2022/12/27 14:12:42 00%
[launcher.Browser]2022/12/27 14:12:43 67%
[launcher.Browser]2022/12/27 14:12:44 100%
[launcher.Browser]2022/12/27 14:12:44 Unzip to: /home/collin/.cache/rod/browser/chromium-1033860
[launcher.Browser]2022/12/27 14:12:44 Progress:
[launcher.Browser]2022/12/27 14:12:44 00%
[launcher.Browser]2022/12/27 14:12:45 32%
[launcher.Browser]2022/12/27 14:12:46 67%
[launcher.Browser]2022/12/27 14:12:46 100%
panic: fork/exec /home/collin/.cache/rod/browser/chromium-1033860/chrome-linux/chrome: no such file or directory

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.glob..func2({0xd866e0?, 0xc0001480c0?})
	github.com/go-rod/rod@v0.112.0/lib/utils/utils.go:60 +0x25
github.com/go-rod/rod/lib/utils.E(...)
	github.com/go-rod/rod@v0.112.0/lib/utils/utils.go:66
github.com/go-rod/rod/lib/launcher.(*Launcher).MustLaunch(0xc00033a000?)
	github.com/go-rod/rod@v0.112.0/lib/launcher/launcher.go:351 +0x67
main.New()
	github.com/charmbracelet/vhs/vhs.go:67 +0x27b
main.Evaluate({0xc0005d2000, 0x864}, {0x145b3a0, 0xc000014018}, {0x0, 0x0, 0x0?})
	github.com/charmbracelet/vhs/evaluator.go:33 +0x1c5
main.glob..func1(0x1a1bee0?, {0xc00047e670, 0x1, 0x1?})
	github.com/charmbracelet/vhs/main.go:60 +0x16d
github.com/spf13/cobra.(*Command).execute(0x1a1bee0, {0xc000024050, 0x1, 0x1})
	github.com/spf13/cobra@v1.6.0/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x1a1bee0)
	github.com/spf13/cobra@v1.6.0/command.go:1040 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.6.0/command.go:968
main.main()
	github.com/charmbracelet/vhs/main.go:135 +0x25
```

### Additional context
This might be due to vhs being outdated in nixpkgs.  #206908 

### Notify maintainers
@maaslalani 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m" - system: `"x86_64-linux"`
 - host os: `Linux 5.15.74, NixOS, 23.05 (Stoat), 23.05.20221217.b48e532`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.12.0`
 - channels(collin): `""`
 - channels(root): `"nixos-21.11.335130.386234e2a61"`
 - nixpkgs: `/nix/store/msv8qjxnga8h81biz33gpa4pyw9kg0j1-source`
```

### PR DESCRIPTION
**Pull request by slinlee**
###### Description of changes

Fixes #208002 by adding the dependency on Chromium. Without this
VHS doesn't work and tries to download Chromium itself which fails
on nix.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by maaslalani — APPROVED** [review]

File: pkgs/applications/misc/vhs/default.nix [hunk 1]
```diff
@@ -14,12 +14,12 @@ buildGoModule rec {
   vendorHash = "sha256-9nkRr5Jh1nbI+XXbPj9KB0ZbLybv5JUVovpB311fO38=";
 
   nativeBuildInputs = [ installShellFiles makeWrapper ];
-  buildInputs = [ ttyd ffmpeg ];
+  buildInputs = [ ttyd ffmpeg chromium ];
```
Comment (NickCao) [inline pkgs/applications/misc/vhs/default.nix:17]: As the programs are only used explicitly in the wrapper below, no need to add them to buildInputs.

Comment (slinlee) [reply pkgs/applications/misc/vhs/default.nix:17]: @NickCao Thanks, I removed them, tested and pushed again.

**Comment by NickCao** [comment]
Please split the changes into two commits
1. vhs: remove redundant buildInputs
2. vhs: add chromium to wrapper

**Comment by slinlee** [comment]
> Please split the changes into two commits
> 
>     1. vhs: remove redundant buildInputs
> 
>     2. vhs: add chromium to wrapper

@NickCao - done!

**Review by NickCao — APPROVED** [review]

**Pull request merged**